frontend/lib/src/dataframes/Quiver.ts

```diff
@@ -299,23 +299,28 @@ export class Quiver {
     return this._data.getChildAt(columnIndex)?.get(rowIndex)
   }
 
+  /**
```

Comment:
These errors are now rendered through the `ErrorBoundary` in `ElementNodeRenderer` which does not support markdown in the message.